roles/sos-vhostuser/files/vhostuser-bind.service

```diff
@@ -1,5 +1,7 @@
 [Unit]
 Description=Vhostuser Interface vfio-pci Bind
+Wants=network-online.target
+After=network-online.target ignition-firstboot-complete.service
```

Comment:
Is this safe? I have seen scenarios when the additional interface dhcp is not available, network-online tends to fail. Is this vhostuser-bind service will be triggered irrespective of success/failure of network-online.target? if yes, then it looks good to me.

Reply:
```
Wants=
A weaker version of Requires=. Units listed in this option will be started if the configuring unit is. However,
if the listed units fail to start or cannot be added to the transaction, this has no impact on the validity of the transaction as a whole.
This is the recommended way to hook start-up of one unit to the start-up of another unit.
```